Bio/ExPASy/Prosite.py

```diff
@@ -300,8 +300,6 @@ def __read(handle):
             record.prorules.extend(rules)
         elif keyword == "DO":
             record.pdoc = value.rstrip(";")
-        elif keyword == "CC":
```

Comment:
In a way it's nice to keep this to be explicit; i.e. we didn't just forget the `CC` case

Reply:
It is already there on line 234, that was the duplication spotted as a likely error.